src/web/components/elements/ImageComponent.tsx

```diff
@@ -94,13 +140,22 @@ export const ImageComponent: React.FC<{
             role={role}
             isMainMedia={isMainMedia}
         >
-            <Picture
-                sources={sources}
-                alt={element.data.alt || ''}
-                src={getFallback(element.imageSources)}
+            <div
+                className={css`
+                    position: relative;
+                `}
             >
-                {children}
-            </Picture>
+                <Picture
```

Comment:
Could the wrapper div, Picture and starRating be pulled into a `React.fc` to avoid repeating them in both places?